### PR TITLE
[Merged by Bors] - expected fix: checkout from master for PR summary

### PR DESCRIPTION
**Pull request by adomani**
This PR accompanies #13602: it makes the script pull code from master, now that it is present there, rather than from the branch that introduced the code.

It is expected to fail CI until the PR on which it depends is merged.

It also fixes a copy-paste error in the labeling of `Diff of declarations`.

---

- [ ] depends on: #13602, where the code is introduced

[Open in Gitpod](https://gitpod.io/from-referrer/)

### CODE REVIEW

File: .github/workflows/PR_summary.yml [hunk 1]
```diff
@@ -41,10 +41,8 @@ jobs:
 
         # Checkout the merge base
         git checkout "$(git merge-base master ${{ github.sha }})"
-        # git checkout origin/master scripts/count-trans-deps.py scripts/import-graph-report.py
-        # get the files that we need from the current PR
-        # TODO ***change*** to `origin/master` in the line below, when merging or right after
-        git checkout origin/adomani/move_decls_autocomment scripts/update_PR_comment.sh
+        # get the files that we need from master
+        git checkout origin/master scripts/update_PR_comment.sh scripts/count-trans-deps.py scripts/import-graph-report.py
```
Comment (adomani) [inline .github/workflows/PR_summary.yml:45]:
```suggestion
```
Can these lines be removed entirely?  If a PR triggers the bot, it should have merged a version of master that contains the scripts anyway.  Will Git use an earlier master for its comparisons?

Comment (jcommelin) [reply .github/workflows/PR_summary.yml:45]: I think you are right.

**Comment by jcommelin** [comment]
bors d+

**Pull request closed without merging**